integration-tests/emulated/src/tests/peregrine/teleportation.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{
+	mock::{
+		network::MockNetworkRococo,
+		para_chains::{AssetHubRococo, Peregrine},
+		relay_chains::Rococo,
+	},
+	utils::get_account_id_from_seed,
+};
+use frame_support::assert_err;
+use frame_support::dispatch::RawOrigin;
+use integration_tests_common::{asset_hub_polkadot, polkadot::ED, ALICE, BOB};
+use peregrine_runtime::PolkadotXcm as PeregrineXcm;
+use sp_core::sr25519;
+use sp_runtime::{DispatchError, ModuleError};
+use xcm::v3::WeightLimit;
+use xcm_emulator::{Here, Junction, Junctions, ParentThen, TestExt, X1};
+
+#[test]
+fn test_teleport_asset_from_regular_peregrine_account_to_asset_hub() {
+	MockNetworkRococo::reset();
+
+	let alice_account_id = get_account_id_from_seed::<sr25519::Public>(ALICE);
+	let bob_account_id = get_account_id_from_seed::<sr25519::Public>(BOB);
+
+	Peregrine::execute_with(|| {
+		assert_err!(
```

Comment:
This is not an `assert_noop!`?

Reply:
[7f1016c](https://github.com/KILTprotocol/kilt-node/pull/603/commits/7f1016cec4f75e682cccf3c5a78d16f5948bdd1a)